entries/val.xml

```diff
@@ -130,6 +130,9 @@ $( "input:text.items" ).val(function( index, value ) {
 });
       </code></pre>
       <p>This example appends the string " items" to the text inputs' values.</p>
+        <div class="warning">
+          <p><strong>Note: </strong> changing the value of an input element using this method won't fire the <code><a href="/change">change</a></code> event.</p>
```

Comment:
I agree with the point that having the same note on multiple places isn't maintainable.

> whether it's needed here since someone should be looking to event docs if they're listening for an event

I do think a note can't hurt. People will be at least on one of these pages when something goes wrong. In fact the [original ticket for this](http://bugs.jquery.com/ticket/14171) proves that, it had the title _".val(var) does not trigger .change"_